### PR TITLE
Adding new examples to Falcon for issue #45

### PR DESCRIPTION
**Pull request by wjin33**

### CODE REVIEW

**Comment by loganharbour** [comment]
@dschwen, @rpodgorney is on to you now

**Comment by rpodgorney** [comment]
@dschwen -- what did you do?  Everything is broken!!!!!!

File: doc/content/examples/rtes_ht.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# HT-coupled RTES Benchmark between FALCON and TOUGH
+
+## Problem Statement
+
+Reservoir Thermal Energy Storage (RTES), also called Geological Thermal Energy Storage (GeoTES) or Aquifer Thermal Energy Storage (ATES), is the concept of injecting hot fluid into the subsurface formation for energy storage and extracting it for later use. This example demonstartes the MOOSE-based FALCON can obtain the same quantitative results as compared to the [TOUGH3](https://tough.lbl.gov/software/new-release-tough3/) code for the heat transfer and fluid flow coupled problems.
```
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:5]: Please replace

This example demonstartes the MOOSE-based FALCON can obtain the same quantitative results as compared to the [TOUGH3](https://tough.lbl.gov/software/new-release-tough3/) code for the heat transfer and fluid flow coupled problems.

with

This example presents a comparison of results obtained using FALCON to those from the [TOUGH3](https://tough.lbl.gov/software/new-release-tough3/) code for the heat transfer and fluid flow coupled problems.

**Review by rpodgorney — CHANGES_REQUESTED** [review]
TOUGH misspelled

**Review by rpodgorney — CHANGES_REQUESTED** [review]
TOUGH misspelled

File: doc/content/examples/rtes_ht.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+!media media/TOUGH_benchmark.png
+  id=fig:benchmark
+  style=width:100%;
+  caption= Model set up comparison between TOUGH and FALCON for the Lower Tuscaloosa formation.
+
+The Lower Tuscaloosa formation located at the Cranfield site was chosen as the storage site. [fig:benchmark] shows the geometry and boundary conditions for the two models built for the TOUGH and FALCON solvers, respectively. The formation thickness is about 24 m and is located at a depth of 3000 m.  Given the symmetry of the 5-spot well configuration, a 1/8 TOUGH model and a quarter FALCON model were built. Note the TOUGH is a finite difference method (FDM) based solver while FALCON is a finite element method based solver, the quarter FALCON model has a better convergence rate. The cap-/bed-rock layers of the TOUH model were replaced by undrained flow boundary conditions and an analytical solution for heat conduction. While cap-/bed-rocks were explicitly simulated in the FALCON model with both no-flux of heat and fluid applied to the top and bottom surfaces. All the side surfaces for both the FALCON and TOUGH models are symmetrical boundaries.
```
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:12]: TOUGH misspelled

File: doc/content/examples/rtes_ht.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+A typical RTES operation of injection-storage-extraction-rest seasonal cycles were simulated. During the injection season, we inject hot fluid with a constant injection temperature of 180 degree  into the injection well and extract fluid from the production well at the same fixed rate of 3 kg/s. During the extraction season, we inject cold fluid with the same temperature as the formation initial temperature T0 = 126 degree C into the injection well and extract fluid from the production well at the same fixed rate of 3 kg/s. No fluid injection or extraction is conducted during the storage and rest seasons. For this problem, we detail the the input file of FALCON int the following section, and the input file for TOUGH3 can be downloaded [here](examples/rtes_benchmark/tough.inp).
```
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:14]: degree should be "degrees C"

File: doc/content/examples/rtes_ht.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+!listing examples/rtes_benchmark/Cranfield_HT_season_caps_water_oldboundary_coarse.i block=Materials
+
+The blue dotted lines show the FALCON prediction with updated input file. The pore pressure at both the injection and extraction wells match each other perfectly. However, the yielded temperature prediction at the extraction well still has significant discrepancy during the storage season as shown in [fig:comparison]. With a careful examination of the two input files, we found this is due to the boundary condition setup of the two wells. The TOUGH solver is based on FDM, and the two wells were realized by two columns of elements with extremly high permeability. While, line sinks were used for FALCON models. Following the same boundary treatment as the TOUGH model, two columns of elements were selcted as different blocks as compared to the formation. The material input for these two blocks are
```
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:26]:
```suggestion
The blue dotted lines show the FALCON prediction with updated input file. The pore pressure at both the injection and extraction wells match each other perfectly. However, the temperature prediction at the extraction well still has significant discrepancy during the storage season as shown in [fig:comparison]. With a careful examination of the two input files, we found this is due to the boundary condition setup of the two wells. The TOUGH solver is based on FDM, and the two wells were realized by two columns of elements with extremly high permeability. While line sinks were used for the FALCON simulations. Following the same boundary treatment as the TOUGH model, two columns of elements were selcted as different blocks as compared to the formation. The material input for these two blocks are 
```

File: doc/content/examples/rtes_ht.md (hunk 1 above)
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:5]:
```suggestion
Reservoir Thermal Energy Storage (RTES), also called Geological Thermal Energy Storage (GeoTES) or Aquifer Thermal Energy Storage (ATES), is the concept of injecting hot fluid into the subsurface formation for energy storage and extracting it for later use. This example provides a comparison between FALCON and the [TOUGH3](https://tough.lbl.gov/software/new-release-tough3/) code for the heat transfer and fluid flow coupled problems and can be used as an example for TOUGH users on how to setup FALCON simulations
```

File: doc/content/examples/rtes_ht.md (hunk 2 above)
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:12]:
```suggestion
The Lower Tuscaloosa formation located at the Cranfield site was chosen as the storage site. [fig:benchmark] shows the geometry and boundary conditions for the two models built for the TOUGH and FALCON solvers, respectively. The formation thickness is about 24 m and is located at a depth of 3000 m.  Given the symmetry of the 5-spot well configuration, a 1/8 TOUGH model and a quarter FALCON model were built. Note the TOUGH is a finite difference method (FDM) based solver while FALCON is a finite element method based solver, the quarter FALCON model has a better convergence rate. The cap-/bed-rock layers of the TOUGH model were replaced by undrained flow boundary conditions and an analytical solution for heat conduction. While cap-/bed-rocks were explicitly simulated in the FALCON model with both no-flux of heat and fluid applied to the top and bottom surfaces. All the side surfaces for both the FALCON and TOUGH models are symmetrical boundaries. 
```

File: doc/content/examples/rtes_ht.md (hunk 3 above)
Comment (rpodgorney) [inline doc/content/examples/rtes_ht.md:14]:
```suggestion
A typical RTES operation of injection-storage-extraction-rest seasonal cycles were simulated. During the injection season, we inject hot fluid with a constant injection temperature of 180 degrees C  into the injection well and extract fluid from the production well at the same fixed rate of 3 kg/s. During the extraction season, we inject cold fluid with the same temperature as the formation initial temperature To = 126 degree C into the injection well and extract fluid from the production well at the same fixed rate of 3 kg/s. No fluid injection or extraction is conducted during the storage and rest seasons. For this problem, we detail the the input file of FALCON int the following section, and the input file for TOUGH3 can be downloaded [here](examples/rtes_benchmark/tough.inp). 
```

File: doc/content/examples/forge_falcon.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
+# Utah FORGE Native State Model
+
+## Problem Statement
+
+The Frontier Observatory for Research in Geothermal Energy (FORGE) site is a multi-year initiative funded by the U.S. Department of Energy for enhanced geothermal system research and development. This example demonstartes that the MOOSE-based FALCON can simulate the native state distribution (i.e., pore pressure, stress, and temperature) of the FORGE site with various in-situ data constraints. A detailed description can be found in [!cite](liu2022forge).
```
Comment (rpodgorney) [inline doc/content/examples/forge_falcon.md:5]:
```suggestion
The Frontier Observatory for Research in Geothermal Energy (FORGE) site is a multi-year initiative funded by the U.S. Department of Energy for enhanced geothermal system research and development. This example presents the development of the native state simulation (i.e., pore pressure, stress, and temperature) of the FORGE site using material property values obtained from the site characterization activities. A detailed description can be found in [!cite](liu2022forge).
```

File: doc/content/examples/mandel.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# Mandel's consolidation
```
Comment (rpodgorney) [inline doc/content/examples/mandel.md:1]: This example is incomplete.  We can allow it to merge now--and I can push @AhmedAlmetwally and @lynnmunday to add more detail

Comment (wjin33) [reply doc/content/examples/mandel.md:1]: Let us merge this first and Either those folks or me can update the rest

Comment (lynnmunday) [reply doc/content/examples/mandel.md:1]: That is good.

**Comment by rpodgorney** [comment]
We'll re-review and make any final changes with the next PR

**Pull request merged**